sor-api/src/main/java/com/bazaarvoice/emodb/sor/api/CompactionControlSource.java

```diff
@@ -16,7 +18,7 @@
 
     StashRunTimeInfo getStashTime(String id, String dataCenter);
 
-    Map<String, StashRunTimeInfo> getAllStashTimes();
+    Map<Pair, StashRunTimeInfo> getAllStashTimes();
```

Comment:
Is it possible to declare as `Pair<String, String>`?

Reply:
Perhaps a good replacement could be a guava [table]?(https://google.github.io/guava/releases/16.0/api/docs/com/google/common/collect/Table.html)